homework/index.js

```diff
@@ -30,17 +33,189 @@
+      title: 'Updated',
+      value: updatedDate,
+    });
+
+    createRepositoryWidget(leftColumn, {
+      faClasses: 'fas fa-code-branch',
+      title: 'Forks',
+      value: repositoriesData.forks,
+    });
+  }
+
+  function appendRepositoriesToSelect(repositoriesData, cb) {
+    const selectList = document.getElementById('repo-select');
+    repositoriesData
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach((repository, index) => {
+        createAndAppend('option', selectList, {
+          text: repository.name,
+          value: index,
+        });
+      });
+
+    // dummy values
+    cb(repositoriesData[0], repositoriesData[0].contributors_url);
+
+    // create an eventlistener for select list options
+    selectList.addEventListener('change', () => {
+      selectList.parentElement.nextElementSibling.innerHTML = '';
+      cb(repositoriesData[selectList.value], repositoriesData[selectList.value].contributors_url);
+    });
+  }
+
+  function fetchRepositoriesAndContributors(url) {
+    fetchJSON(url, (errRepository, repositoriesData) => {
+      if (errRepository) {
+        createAndAppend('div', root, {
+          text: errRepository.message,
+          class: 'alert alert-danger display-3 text-center',
+        });
         return;
       }
-      createAndAppend('pre', root, { text: JSON.stringify(repositories, null, 2) });
+
+      appendRepositoriesToSelect(repositoriesData, (repositoryObj, contributorURL) => {
+        createRepositoryLayout(repositoryObj);
+        fetchJSON(contributorURL, (errContributors, contributorData) => {
+          if (errContributors) {
+            createAndAppend('div', root, {
+              text: `${errContributors.message}... The data cannot be fetched`,
+              class: 'alert alert-danger display-3 text-center',
+            });
+            return;
+          }
+          createContributorsLayout(contributorData);
+        });
+      });
     });
   }
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  function createHeader() {
+    const header = createAndAppend('header', root, {
+      class: 'container jumbotron mt-3 mb-3 px-5',
+    });
+    createAndAppend('main', root, { id: 'main', class: 'container align-items-start' });
+    const headerDiv = createAndAppend('div', header, {
+      class: 'header-group d-flex flex-row justify-content-between mb-5 px-5',
+    });
+    createAndAppend('h2', headerDiv, {
+      text: 'Hack Your Future Repositories',
+      class: 'display-4 d-flex align-items-center mx-auto col-sm header-title',
+    });
+    const hyfLink = createAndAppend('a', headerDiv, {
+      href: 'https://www.hackyourfuture.net/',
+      target: '_blank',
+    });
+    createAndAppend('img', hyfLink, {
+      class: 'img-fluid rounded img-thumbnail mx-auto',
+      src: './hyf.png',
+      alt: 'hack your future thumbnail',
+    });
+    const selectList = createAndAppend('select', header, {
```

Comment:
Now that you have removed the default option, the `selectList` variable is no longer needed. ESLint flags it as unused, causing the check by Travis for your PR to fail. Can you fix this final thing?

Reply:
Yes, i didnt see them on the logs. I noticed now.